tests/openbis_functions/test_openbis_db.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+from requests import patch
+
+from lebedigital.openbis.expstep import ExpStep
+from pybis import Openbis
+import pandas as pd
+
+# https://docs.pytest.org/en/7.2.x/how-to/monkeypatch.html
+
+
+class MockResponseGetSampleNames:
+
+    # MockResponse needs the .df attribute as openbis methods need the .df to get the dataframe instead of a
+    # pybis object
+
+    def __init__(self) -> None:
+        self.df = self.get_samples()
```

Comment:
The df is there because i run the get_samples().df method in my code and without it the mocking object threw an error. The problem here was that pybis returns a "Things" object which would be hard for me to simulate in the mocked tests and i use the dataframe conversion of it in the code anyways.